packages/launcher/package.json

```
@@ -13,7 +13,7 @@
     "clean": "node scripts/clean.js || true",
     "clean-deps": "rm -rf node_modules",
     "clean-js": "npm run clean",
-    "size": "t=\"$(npm pack .)\"; wc -c \"${t}\"; tar tvf \"${t}\"; rm \"${t}\";",
+    "size": "cross-env t=\"$(npm pack .)\"; wc -c \"${t}\"; tar tvf \"${t}\"; rm \"${t}\";",
```

same thing - can be simplified to
```suggestion
    "size": "npm pack --dry",
```